From 57ff49bc924ded5c9fcfcb60305a1c5200064c04 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Wed, 14 Mar 2012 19:53:22 +0200
Subject: [PATCH 063/311] s/path->/path[0]./

If this sets the property of a particular element of an array,
`arr[0].prop = val' expression is better than `arr->prop = val'.

Reported-by: Nikita Danilov <nikita_danilov@xyratex.com>
Reviewed-on: http://reviewboard.clusterstor.com/r/714/
---
 conf/confc.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/conf/confc.h b/conf/confc.h
index 363d9b1..2f6ec8e 100644
--- a/conf/confc.h
+++ b/conf/confc.h
@@ -246,7 +246,7 @@
  *         if (svc->cs_type != svc_type)
  *             continue;
  *
- *         path->pc_u.relation = C2_CO_NODE;
+ *         path[0].pc_u.relation = C2_CO_NODE;
  *         node = c2_confc_open_sync(svc, path, 1); // svc/node
  *         if (node == NULL)
  *             goto end;
@@ -256,7 +256,7 @@
  *             continue;
  *         }
  *
- *         path->pc_u.relation = C2_CO_NIC;
+ *         path[0].pc_u.relation = C2_CO_NIC;
  *         d = c2_confc_diropen_sync(node, path, 1); // node/nics
  *         if (d == NULL) {
  *             c2_confc_close(node);
@@ -267,7 +267,7 @@
  *         }
  *         c2_confc_dirclose(d);
  *
- *         path->pc_u.relation = C2_CO_SDEV;
+ *         path[0].pc_u.relation = C2_CO_SDEV;
  *         d = c2_confc_diropen_sync(node, path, 1); // node/sdevs
  *         if (d == NULL) {
  *             c2_confc_close(node);
-- 
1.8.3.2

